internal/store/postgres/asset_repository.go

```diff
@@ -453,9 +453,11 @@ func (r *AssetRepository) deleteWithPredicate(ctx context.Context, pred sq.Eq) (
 
 func (r *AssetRepository) insert(ctx context.Context, ast *asset.Asset) (id string, err error) {
 	err = r.client.RunWithinTx(ctx, func(tx *sqlx.Tx) error {
+		ast.CreatedAt = time.Now()
+		ast.UpdatedAt = ast.CreatedAt
 		query, args, err := sq.Insert("assets").
-			Columns("urn", "type", "service", "name", "description", "data", "url", "labels", "updated_by", "version").
-			Values(ast.URN, ast.Type, ast.Service, ast.Name, ast.Description, ast.Data, ast.URL, ast.Labels, ast.UpdatedBy.ID, asset.BaseVersion).
+			Columns("urn", "type", "service", "name", "description", "data", "url", "labels", "created_at", "updated_by", "updated_at", "version").
```

Comment:
QQ, doesn't the `assets` table already have default value?

https://github.com/goto/compass/blob/4c21024ab2b748d84be5caa3e1350b3cb87a938b/internal/store/postgres/migrations/000003_create_assets_table.up.sql#L12-L13

Reply:
It gets populated in the database only for assets table. We pass the same asset to different function for creating a row in `assets_versions` table and later for indexing into Elasticsearch. If we don't populate the fields here, those instances will not have the correct timestamps.

Reply:
noted